catalog/transformer/catalog-transformer-shapefile/src/main/java/ddf/catalog/transformer/shape/file/transformer/api/ShapeFileAttributes.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.shape.file.transformer.api;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/** This class is used to define extra taxonomy attributes for the ShapeFileInputTransformer. */
+public class ShapeFileAttributes implements ShapeFile, MetacardType {
+  private static final Set<AttributeDescriptor> DESCRIPTORS;
+
+  private static final String NAME = "shapefile";
+
+  static {
+    Set<AttributeDescriptor> descriptors = new HashSet<>();
+    descriptors.add(
+        new AttributeDescriptorImpl(
+            SHAPE_MIN_LAT,
```

Comment:
Why do we define a MetacardType for shape files and add min/max lat/lon attributes?  The location attribute isn't sufficient for this?

Reply:
Shapefiles are typically very complex. They follow a river or the borders of a city. So these attributes make it easier to display a bounding box that encompasses the shape on the 3D view.

Reply:
Hoping we can remove these and try to do a smart approximation of the geo in the shapefile and set it on the location attribute